CHANGELOG.md

```diff
@@ -0,0 +1,39 @@
+Change log
+===
+
+### Newer Versions 
+
+For newer versions. See [GitHub releases](https://github.com/jenkinsci/matlab-plugin/releases)

```

Comment:
For newer versions of the plugin, see GitHub releases.

Question:  Does this mean that the version that the user sees in this repository is not the newest version of the plugin?!

Reply:
Done.